gateway/config/sandbox.lua

```diff
@@ -0,0 +1 @@
+staging.lua
```

Comment:
Why `sandbox` and why it's the same as staging?

Reply:
Because some configurations can still have `THREESCALE_DEPLOYMENT_ENV=sandbox`. It was the original value in AMP 2.0 IIRC. We alias it now to `staging` which makes more sense.

Reply:
:+1: